tests/helpers/deposit.js

```diff
@@ -0,0 +1,28 @@
+const P = require('aigle')
+
+const deposit = async (dvf, token, amount, starkPrivKey) => {
+    try {
+        const depositResponse = await dvf.deposit(token, amount, starkPrivKey)
+
+        await P.retry({ times: 20, interval: 30000 }, () => checkBalance(dvf))
```

Comment:
@thiagokroger can we start checking after 3 minutes instead of every 30 seconds? Because the deposit will not be processed until 2 to 3 minutes after the initial deposit request.

Reply:
@AliMeer good point, I've just increased the time to 2 minutes, that wai we can cover the fast case (2 minutes), the normal case (max 4 min) and so on

Reply:
Great 👍 And same applies for withdrawal as well.

Reply:
right, I've just updated the withdraw too